### PR TITLE
Added tech mapping and using it in TMBA

### PR DESCRIPTION
**Pull request by mkcinu**

### CODE REVIEW

File: mat_dp_pipeline/data_sources/utils.py [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 @cache
 def default_location_mapping() -> dict[str, Path]:
-    countries = pd.read_csv(COUNTRY_MAPPING_CSV)
+    countries = pd.read_csv(COUNTRY_MAPPING_CSV, keep_default_na=True)
```
Comment (mkcinu) [inline mat_dp_pipeline/data_sources/utils.py:11]: This goes away anyway.. ignore

File: mat_dp_pipeline/data_sources/tmba.py [hunk 2]
```diff
@@ -60,8 +48,8 @@ def __call__(self, output_dir: Path) -> None:
         category = targets["parameter"].map(self._parameter_to_category)
         targets.insert(0, "Category", category)
         for pattern, replacement in self._variable_to_specific.items():
-            # Remove (ignore) if None replacement, else replace
-            if replacement is None:
+            # Remove (ignore) if there is no replacement, else replace
+            if not replacement:
```
Comment (mark-todd) [inline mat_dp_pipeline/data_sources/tmba.py:52]: Why the change here?
